core/adapters/compare.go

```diff
@@ -15,6 +15,7 @@ type Compare struct {
 	Value    string `json:"value"`
 }
 
+// Error message
```

Comment:
What is the point of this comment? If we're going to have comments, let's have them be useful please.